doc/manual-installation.md

```diff
@@ -30,6 +30,19 @@ Install the operator.
 $ kubectl create -f deploy/operator.yaml
 ```
 
+> Note:  We can enable testmode on operator by passing an argument `-test` in `operator.yaml` file.This will be useful if we want to run pravega on minikube or cluster with limited resources. In test mode, operator will allow us to run pravega with single bookkeeper instance.
```

Comment:
Please provide a header to this section: "Deploying in Test Mode". Provide a link to the documentation about test mode in Readme.md. 
We do not want to encourage users to use "test mode" as it is supposed to be used only in extreme cases where we have severe resource constraints in the environment where Pravega is being deployed. Lets make that very clear in this notice. Pravega with a single BK, controller and SSS is not recommended and has not been tested extensively. Its just a bare minimum toy setup for Pravega.

Reply:
Fixed